trunk/SUAVE/Methods/Noise/Fidelity_One/Propeller/compute_broadband_noise.py

```diff
@@ -5,119 +5,351 @@
+
+            # ------------------------------------------------------------
+            # ****** TRAILING EDGE BOUNDARY LAYER PROPERTY CALCULATIONS  ******
 
+            delta[i,:,:,:,:,:,0]        = vectorize(lower_surface_delta,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                    # lower surface boundary layer thickness
+            delta[i,:,:,:,:,:,1]        = vectorize(upper_surface_delta,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                    # upper surface boundary layer thickness
+            delta_star[i,:,:,:,:,:,0]   = vectorize(lower_surface_delta_star,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                               # lower surface displacement thickness
+            delta_star[i,:,:,:,:,:,1]   = vectorize(upper_surface_delta_star,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                               # upper surface displacement thickness
+            dp_dx[i,:,:,:,:,:,0]        = vectorize(lower_surface_dp_dx ,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                   # lower surface pressure differential
+            dp_dx[i,:,:,:,:,:,1]        = vectorize(upper_surface_dp_dx ,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                   # upper surface pressure differential
+            Ue[i,:,:,:,:,:,0]           = vectorize(lower_surface_Ue,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                       # lower surface boundary layer edge velocity
+            Ue[i,:,:,:,:,:,1]           = vectorize(upper_surface_Ue,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                       # upper surface boundary layer edge velocity
+            tau_w[i,:,:,:,:,:,0]        = vectorize(lower_surface_cf*(0.5*rho_blade[i]*(U_blade[i]**2)),num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)    # lower surface wall shear stress
+            tau_w[i:,:,:,:,:,:,1]       = vectorize(upper_surface_cf*(0.5*rho_blade[i]*(U_blade[i]**2)),num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)    # upper surface wall shear stress
+            Theta[i,:,:,:,:,:,0]        = vectorize(lower_surface_theta,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                    # lower surface momentum thickness
+            Theta[i,:,:,:,:,:,1]        = vectorize(upper_surface_theta,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 7)                                    # upper surface momentum thickness
+
+        # Update dimensions for computation
+        r         = vectorize(r,num_cpt,num_mic,num_sec,num_prop,num_azi,num_cf,vectorize_method = 1)
```

Comment:
same comments for the vectorize function as I already flagged in compute_source_coordinates.py; 
The vectorize_method number is really unintuitive; I think it might be better to just use the one-liner directly, ie.  for r:

r = np.repeat(np.repeat(np.repeat(np.repeat(np.repeat(r[None,None,None,:,None,None], num_cpt, axis=0),num_mic,axis=1),num_prop,axis=2),num_azi, axis=4), num_cf, axis=5)

This does the same thing and you don't have to go into vectorize to figure out what vectorize_method 1 means.

Reply:
Removed the vectorize function